tutorials/00-introduction/00_introduction.jmd

```diff
@@ -175,7 +175,8 @@ coinflip(y::AbstractVector{<:Real}) = coinflip(; N=length(y)) | (; y);
 In the Turing model the prior distribution of the variable `p`, the probability of heads in a coin toss, and the distribution of the observations `y` are specified on the right-hand side of the `~` expressions.
 The `@model` macro modifies the body of the Julia function `coinflip` and, e.g., replaces the `~` statements with internal function calls that are used for sampling.
 
-We combine the model with the observations:
+We have also defined a convenience function that makes use of the `condition` syntax (see ['Condition & decondition'](https://beta.turing.ml/DynamicPPL.jl/stable/api/#Condition-and-decondition) in DynamicPPL.jl
```

Comment:
I guess the title is not particularly helpful here. Maybe we could just refer to
```suggestion
We have also defined a convenience function that makes use of the `condition` syntax (see the [documentation of DynamicPPL.jl](https://beta.turing.ml/DynamicPPL.jl/stable/api/#Condition-and-decondition)
```

Reply:
Another general comment is that maybe it would be better to move the code to the next code block so that we can explain the conditioning before it is used.

Reply:
I applied the suggestions, what do you think @PavanChaggar?